testsuite/testcases/src/fullnode_reboot_stress_test.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{LoadDestination, NetworkLoadTest};
+use forge::{NetworkContext, NetworkTest, Result, Swarm, Test};
+use rand::seq::SliceRandom;
+use rand::thread_rng;
+use std::time::Duration;
+use tokio::runtime::Runtime;
+use tokio::time::Instant;
+
+pub struct FullNodeRebootStressTest;
+
+impl Test for FullNodeRebootStressTest {
+    fn name(&self) -> &'static str {
+        "fullnode reboot stress test"
+    }
+}
+
+impl NetworkLoadTest for FullNodeRebootStressTest {
```

Comment:
possible to combine `FullNodeRebootStressTest` and `ValidatorRebootStressTest` logic and just change the reboot targets?

Reply:
I thought about that but its not that clean, so would leave it as is for now.